modules/tinymce/changelog.txt

```diff
@@ -1,5 +1,6 @@
 Version 5.2.1 (TBD)
     Fixed "is decorative" checkbox in the image dialog not staying checked on certain actions #FOAM-11
+    Fixed possible uncaught exception when creating a parser #TINY-4742
```

Comment:
This is pretty vague, can you make it more specific so anyone reading this has a better idea of what changed? Maybe say something like when a style was removed using a parser filter?